devito/dse/transformer.py

```diff
@@ -1,5 +1,6 @@
-from devito.ir.clusters import ClusterGroup, groupby
+from devito.ir.clusters import ClusterGroup, optimize
```

Comment:
Is the 'optimize' a bit vague?

Reply:
maybe yes, but it's pulled from ``clusters`` so it should be clear that it's "optimizing" clusters